code/modules/spells/spell_types/shapeshift.dm

```diff
@@ -14,10 +14,10 @@
 	var/shapeshift_type
 	var/list/current_shapes = list()
 	var/list/current_casters = list()
-	var/list/possible_shapes = list(/mob/living/simple_animal/mouse,\
-		/mob/living/simple_animal/pet/dog/corgi,\
-		/mob/living/simple_animal/hostile/carp/ranged/chaos,\
-		/mob/living/simple_animal/bot/ed209,\
+	var/list/possible_shapes = list(/mob/living/simple_animal/mouse,
+		/mob/living/simple_animal/pet/dog/corgi,
+		/mob/living/simple_animal/hostile/carp/ranged/chaos,
+//		/mob/living/simple_animal/bot/ed209,
```

Comment:
I'm well aware of how Wizard is balanced. and it's made to have an active population of 60+ with a full security team who are able to use their radios and stay together. And also a station that's actually wanting to fight rather than just do the job. 

And the best way to stop shittery, is to nullify the parts that are really shitty.